.github/labeler.yml

```diff
@@ -0,0 +1,43 @@
+# Add/remove 'critical' label if issue contains the words 'urgent' or 'critical'
+critical:
+    - '(critical|urgent)'
+
+cli:
+    - 'cli'
+
+daemon:
+    - 'daemon'
+
+coordinator:
+    - 'coordinator'
+
+runtime:
+    - 'runtime'
+
+python:
+    - 'python'
+
+c:
+    - 'c'
```

Comment:
Doesn't this match on all issues that contain a `c` somewhere (e.g. as part of some other word)?

Reply:
True, ahah, thanks!